client/lib/abtest/active-tests.js

```diff
@@ -41,6 +41,15 @@ module.exports = {
 		},
 		defaultVariation: 'singlePurchaseFlow'
 	},
+	plansDescriptions: {
```

Comment:
Can we name this a bit differently? E.g.: plansDescriptionsOnHover. I think `planDescriptions` is too vague. :)